mm2src/coins/utxo/utxo_tests.rs

```diff
@@ -3354,7 +3354,6 @@ fn test_tx_details_kmd_rewards() {
 /// then `TransactionDetails::kmd_rewards` has to be `Some(0)`, not `None`.
 /// https://kmdexplorer.io/tx/f09e8894959e74c1e727ffa5a753a30bf2dc6d5d677cc1f24b7ee5bb64e32c7d
 #[test]
-#[ignore]
```

Comment:
This test is called `test_tx_details_kmd_rewards_claimed_by_other` but we have claimed by me always true, should we keep it ignored? maybe rename it instead and write a note about `claimed_by_me`, it can be named `test_tx_details_kmd_rewards_amount_in_nonself_spend`. This is related to the first comment, if `tx_details_by_hash` is fixed and we allow `claimed_by_me` to be false, all these tests can be un-ignored even the deleted one.